ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -7,37 +7,62 @@ import UIKit
 
 /**
  `CommandBar` is a horizontal scrollable list of icon buttons divided by groups.
- Provide `itemGroups` in `init` to set the buttons in the scrollable area. Optional `leadingItem` and `trailingItem` add fixed buttons in leading and trailing positions. Each `CommandBarItem` will be represented as a button.
+ Provide `itemGroups` in `init` to set the buttons in the scrollable area. Optional `leadingItemGroups` and `trailingItemGroups` add button in leading and trailing positions. Each `CommandBarItem` will be represented as a button.
  Set the `delegate` property to determine whether a button can be selected and deselected, and listen to selection changes.
  */
 @objc(MSFCommandBar)
 open class CommandBar: UIView {
     // Hierarchy:
     //
-    // leadingButton
+    // leadingCommandGroupsStackView
+    // |--buttons
     // containerView
     // |--layer.mask -> containerMaskLayer (fill containerView)
     // |--subviews
     // |  |--scrollView (fill containerView)
     // |  |  |--subviews
     // |  |  |  |--stackView
     // |  |  |  |  |--buttons (fill scrollView content)
-    // trailingButton
+    // trailingCommandGroupsStackView
+    // |--buttons
 
     // MARK: - Public methods
 
+    @available(*, renamed: "init(itemGroups:leadingItemGroups:trailingItemGroups:)")
     @objc public init(itemGroups: [CommandBarItemGroup], leadingItem: CommandBarItem? = nil, trailingItem: CommandBarItem? = nil) {
         self.itemGroups = itemGroups
 
-        super.init(frame: .zero)
-
         if let leadingItem = leadingItem {
-            self.leadingButton = button(forItem: leadingItem, isPersistSelection: false)
+            self.leadingItemGroups = [[leadingItem]]
         }
+
         if let trailingItem = trailingItem {
-            self.trailingButton = button(forItem: trailingItem, isPersistSelection: false)
+            self.trailingItemGroups = [[trailingItem]]
         }
 
+        self.leadingCommandGroupsStackView = CommandBarCommandGroupsStackView(itemGroups: self.leadingItemGroups)
+        self.centerCommandGroupsStackView = CommandBarCommandGroupsStackView(itemGroups: self.itemGroups)
```

Comment:
The term "center" feels off, open to naming suggestions. I feel that providing some sort of position in the naming is important.

Reply:
How about "middle" or "main"? I agree that center is a little bit off because it's not necessarily centered.

Reply:
"main" was at the top of my list so I'll go with that.